api/handler/service_batch_operation.go

```diff
@@ -296,6 +281,14 @@ func (b *BatchOperationHandler) createEvents(tenantID, operator string, batchOpR
 			SynType:   dbmodel.ASYNEVENTTYPE,
 			OptType:   req.OpType(),
 		}
+		_, ok := bads[req.GetEventID()]
+		if ok {
+			event.Reason = memoryType
```

Comment:
This needs to be translated into the user language, right?

Reply:
UI will translate it.